content/en/blog/_posts/2016-12-00-Statefulset-Run-Scale-Stateful-Applications-In-Kubernetes.md

```diff
@@ -297,7 +297,7 @@ zk-0      0/1       Terminating   0         15m
 
 
 
-You can use [kubectl apply](/docs/user-guide/kubectl/kubectl_apply/) to recreate the zk StatefulSet and redeploy the ensemble.
+You can use [kubectl apply](/docs/user-guide/kubectl/kubectl_apply) to recreate the zk StatefulSet and redeploy the ensemble.
```

Comment:
```suggestion
You can use [kubectl apply](/docs/reference/generated/kubectl/kubectl-commands#apply) to recreate the zk StatefulSet and redeploy the ensemble.
```

Reply:
@jihoon-seo All the links does redirect to all these target URL right. Is there any reason that we want to put the direct target URL?

Reply:
@niteshseram
Here I quote @tengqm's [comment](https://github.com/kubernetes/website/issues/28575#issuecomment-866671339):
> The correct fix should be avoid using old URIs when page are moved around.